deploy/charts/rook-ceph-cluster/templates/cephblockpool.yaml

```diff
@@ -19,7 +19,15 @@ provisioner: {{ $root.Values.operatorNamespace }}.rbd.csi.ceph.com
 parameters:
   pool: {{ $blockpool.name }}
   clusterID: {{ $root.Release.Namespace }}
+  csi.storage.k8s.io/provisioner-secret-name: rook-csi-rbd-provisioner
```

Comment:
The helm upgrade test is failing with this error: 
```
Error: UPGRADE FAILED: cannot patch "ceph-block-test-sc" with kind StorageClass: StorageClass.storage.k8s.io "ceph-block-test-sc" is invalid: parameters: Forbidden: updates to parameters are forbidden. && cannot patch "ceph-filesystem-test-sc" with kind StorageClass: StorageClass.storage.k8s.io "ceph-filesystem-test-sc" is invalid: parameters: Forbidden: updates to parameters are forbidden. , err -> exit status 1
```

There must be a difference in the storage class parameters when they moved from the values to the template. Or else maybe we just can't touch these in the existing storage class template, and we will only be able to follow this new pattern for the new snapshot storage class.

Reply:
i still need to figure out the reason for that one. will send follow-up for storageclass. reverted storageclass changes.